incubator-tools/combine_address_line/combine_address_line.ipynb

```diff
@@ -0,0 +1,546 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.    def get_coordinate(entity: documentai.Document.Entity, arg: str) -&gt; int:
</pre><p>This is incredibly inefficient. This function should return all of the entity coordinates, then the caller can decide which to use.</p><p> </p><p>I also think there's already a utility function for this.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Refactored code.